FSharp.Profiles.props

```diff
@@ -2,12 +2,12 @@
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Project>
 
-  <PropertyGroup Condition="$(TargetFramework.StartsWith('net4'))">
+  <PropertyGroup Condition="'$(TargetFrameworkName)' == '.NETFramework'">
     <DefineConstants Condition="'$(MonoPackaging)' == 'true'">$(DefineConstants);CROSS_PLATFORM_COMPILER</DefineConstants>
     <DefineConstants>$(DefineConstants);ENABLE_MONO_SUPPORT</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$(TargetFramework.StartsWith('netstandard')) OR $(TargetFramework.StartsWith('netcoreapp'))">
+  <PropertyGroup Condition="'$(TargetFrameworkName)' == '.NETStandard' OR '$(TargetFrameworkName)' == '.NETCoreApp' OR '$(TargetFrameworkName)' == '.NET'">
```

Comment:
I was under the impression `TargetFrameworkIdentifier` (which I think you meant to use here) will still be `.NETCoreApp` even for .NET 5 - cc @sfoslund

Reply:
Yes, TargetFrameworkIdentifier will still be .NETCoreApp, but if you are parsing the beginning of the TargetFramework it will just be net5.0.

Reply:
Oh, I confused the property name. And I think I had seen somewhere .NET 5+ referred to as .NET but don't remember...